docs/chapter_appendix/contribution.md

```diff
@@ -43,3 +43,20 @@ docker-compose up -d
 ```shell
 docker-compose down
 ```
+
+## 本机开发部署 
+在任何一个支持python3.9以上的平台上都兼容。  这里用linux举例（喜欢干净环境的的可以使用virtualenv隔离）   
+**使用该方法的部署的优势是编辑后可以刷新实时预览**
+```bash
+cd hello-algo
+pip3 install  --upgrade pip
+pip3 install -r requirements.txt
```

Comment:
I believe creating `requirements.txt` is more standard. However, hello-algo only depends on the mkdocs. It is not necessary to create a `requirements.txt`